stix/idl/processing/aux_data/stx_create_auxiliary_data.pro

```diff
@@ -0,0 +1,197 @@
+;+
+;
+; name:
+;       stx_read_aux_fits
+;
+; :description:
+;    Read an auxiliary fits file an returns the values of the STIX pointing coordinates, apparent radius of the Sun, 
+;    roll angle, pitch, yaw, L0 and B0 for a considered time range
+;
+; :params:
+;    fits_path : in, required, type="string"
+;                the path of the FITS file to be read. Passed through to mrdfits.
+;                
+;    time_range: in, required, type="string array"
+;                array containing the UTC start time and end time of the considered time range      
+;
+; :keywords:
+;    use_sas: if set, avoid check on the reliability and use SAS solution for providing an estimate of STIX pointing 
+;    dont_use_sas: if set, bypass SAS solution and use spacecraft pointing (corrected for systematics) instead
+;    silent: if set, don't dieplay information regarding pointing
+;
+; :returns:
+;
+;    a structure containing the average values of the STIX pointing estimate, 
+;    apparent radius of the Sun, roll angle, L0 and B0 
+;    computed over a considered time range
+;
+; :examples:
+;
+;    aux_data = stx_create_auxiliary_data(fits_path, time_range)
+;
+; :history:
+;
+;    May 1, 2022: Massa P. (MIDA, Unige), created
+;    2022-08-24, F. Schuller (AIP, Germany): implemented interpolation if no measurement found in time range
+;    2022-09-09, FSc: added optional argument 'dont_use_sas'
+;    2022-09-23, FSc: keyword 'silent' added; if not set, now displays messages about the pointing correction used
+;    2022-09-28, FSc: displays a warning if dispersion in pointing > 3 arcsec
+;-
+function stx_create_auxiliary_data, fits_path, time_range, use_sas=use_sas, dont_use_sas=dont_use_sas, silent=silent
+
+  default, use_sas, 0
+  default, dont_use_sas, 0
```

Comment:
Sure, I can rename these keywords as you suggest if it sounds clearer.
I just tested, if both are sets, then `dont_use_sas` wins. We could display a warning since it should not happen to set both.